scripts/asm-installer/asm_vm

```diff
@@ -631,6 +646,10 @@ EOF
   elif [[ -n "${KEY_FILE}" ]]; then
     fatal "Couldn't find key file ${KEY_FILE}."
   fi
+
+  if [[ ! -z "${SOURCE_INSTANCE_TEMPLATE}" ]] && [[ ! -z "${IMAGE_FAMILY}" ]]; then
+      warn "source instance template has been specified and the image family parameter will not be used"
```

Comment:
The description says having two flags specified together is not valid. If it is a validation error, we should error out.

Reply:
ok I will change it to a fatal